README.md

```diff
@@ -125,3 +123,4 @@ See [CONTRIBUTING.md](.github/CONTRIBUTING.md).
 [sentry release]: https://docs.sentry.io/platforms/python/configuration/releases/
 [data source name]: https://docs.sentry.io/product/sentry-basics/dsn-explainer/
 [GitHub Container Registry]: https://github.com/orgs/python-discord/packages/container/package/snekbox
+[`NsJail`]: snekbox/nsjail.py
```

Comment:
I don't think this linking worked correctly. I was not able to press `NsJail` and get linked to this file.

Reply:
[55270d2](https://github.com/python-discord/snekbox/pull/145/commits/55270d29dac5c88f8cd70324b0febb6c792b05e4)

Reply:
Sorry, I made the change but forgot to copy it over to the file git was tracking.

Reply:
c9bfff8